x-pack/plugins/observability_log_explorer/common/constants.ts

```diff
@@ -6,3 +6,5 @@
  */
 
 export const OBSERVABILITY_LOG_EXPLORER_APP_ID = 'observability-log-explorer';
+export const OBSERVABILITY_OVERVIEW_APP_ID = 'observability-overview';
```

Comment:
**suggestion**: These IDs are defined [here](https://github.com/elastic/kibana/blob/main/packages/deeplinks/observability/constants.ts) by the `@kbn/deeplinks-observability` package, apart from importing those from there it might be a good opportunity to also replace `OBSERVABILITY_LOG_EXPLORER_APP_ID` import and remove this duplicate definition.